integration/pvtdatapurge/data_purge_test.go

```diff
@@ -301,136 +254,106 @@ var _ = Describe("Pvtdata purge", func() {
+			// restart original peers
+			startPeer(network, processes, peerRunners, org1Peer0)
+			network.Peers = append(network.Peers, org1Peer0)
 
-			ledgerHeight := nwo.GetLedgerHeight(network, network.Peers[0], channelID)
+			startPeer(network, processes, peerRunners, org2Peer0)
+			network.Peers = append(network.Peers, org2Peer0)
 
-			for _, peer := range network.Peers {
-				stopPeer(network, processes, peer)
-			}
-			stoppedPeers := network.Peers
-			network.Peers = []*nwo.Peer{}
+			startPeer(network, processes, peerRunners, org3Peer0)
+			network.Peers = append(network.Peers, org3Peer0)
 
-			startNewPeer(network, orderer, org3Peer1, ledgerHeight, processes, peerRunners)
+			// Wait for reconciliation to complete and then check for reconciled data
+			time.Sleep(30 * time.Second)
+			marblechaincodeutil.AssertPresentInCollectionM(network, channelID, chaincode.Name, `test-marble-8-keep`, org3Peer1)
+			marblechaincodeutil.AssertPresentInCollectionMPD(network, channelID, chaincode.Name, `test-marble-8-keep`, org3Peer1)
+			marblechaincodeutil.AssertDoesNotExistInCollectionM(network, channelID, chaincode.Name, `test-marble-9-purge`, org3Peer1)
+			marblechaincodeutil.AssertDoesNotExistInCollectionMPD(network, channelID, chaincode.Name, `test-marble-9-purge`, org3Peer1)
 
-			nwo.PackageAndInstallChaincode(network, *chaincode, org3Peer1)
-			network.VerifyMembership([]*nwo.Peer{org3Peer1}, channelID, chaincode.Name)
+			By("Adding two marbles, removing Org3 from the collection, purging the marbles, and confirming that they still got purged from Org3")
+			marblechaincodeutil.AddMarble(network, orderer, channelID, chaincode.Name, `{"name":"test-marble-10-purge-after-ineligible", "color":"white", "size":4, "owner":"liz", "price":4}`, org2Peer0)
+			marblechaincodeutil.AddMarble(network, orderer, channelID, chaincode.Name, `{"name":"test-marble-11-purge-after-ineligible", "color":"orange", "size":80, "owner":"clive", "price":88}`, org2Peer0)
 
-			runner := peerRunners[org3Peer1.ID()]
-			Eventually(runner.Err(), network.EventuallyTimeout).Should(gbytes.Say("Could not fetch \\(or mark to reconcile later\\) 2 eligible collection private write sets for block \\[\\d+\\] \\(0 from local cache, 0 from transient store, 0 from other peers\\)\\. Will commit block with missing private write sets:\\[txID: [0123456789abcdef]+, seq: \\d+, namespace: marblesp, collection: collectionMarblePrivateDetails"))
+			marblechaincodeutil.AssertPresentInCollectionM(network, channelID, chaincode.Name, "test-marble-10-purge-after-ineligible", org2Peer0)
+			marblechaincodeutil.AssertPresentInCollectionMPD(network, channelID, chaincode.Name, "test-marble-10-purge-after-ineligible", org2Peer0)
+			marblechaincodeutil.AssertPresentInCollectionM(network, channelID, chaincode.Name, "test-marble-11-purge-after-ineligible", org2Peer0)
+			marblechaincodeutil.AssertPresentInCollectionMPD(network, channelID, chaincode.Name, "test-marble-11-purge-after-ineligible", org2Peer0)
```

Comment:
As the main test here is to check the deletion on `Org3` peers, it would be better to test the presence on `Org3` peers here.